File: Java/LinearSearch.java
public class LinearSearch{  
    public static void main(String a[]) {    
        int[] a1= {50, 40, 20, 30, 70, 60};    
        int x = 70;   
        int result = linearSearch(a1, x);
        if(result == -1)
            System.out.println("Element is not present in the array.");
        else
            System.out.println(x+" is present in the array at index: "+ result);    
    } 

    public static int linearSearch(int[] arr, int key) {    
        for(int i=0;i<arr.length;i++) {    
            if(arr[i] == key) {    
                return i;    
            }    
        }    
        return -1;    
    }       
}

File: CPP/Linear_Search.cpp
//prograam to perform linear search
#include<stdio.h>
int main()
{
	int n,arr[100],i,search,flag=0;
	printf("Enter the size of the array \n");
	scanf("%d",&n);
	printf("Enter  the array \n");
	for (i=0;i<n;i++)
	{
		scanf("%d",&arr[i]);
	}
	printf("Enter the no. to searched \n");
	scanf("%d",&search);
	for(i=0;i<n;i++)
	{
		if(search == arr[i])
		{
			flag =1;
			break;
		}
	}
	if (flag == 1)
	{
		printf("Search is successful and element is present at %d \n ",i+1);
	}
	else
	{
		printf("Sorry element not found \n");
	}
}
